packages/world/ts/config/v2/defaults.ts

```diff
@@ -26,6 +26,8 @@ export type MODULE_DEFAULTS = typeof MODULE_DEFAULTS;
 export const CODEGEN_DEFAULTS = {
   worldInterfaceName: "IWorld",
   worldgenDirectory: "world",
+  systemLibrariesDirectory: "libraries",
```

Comment:
imo this should be in 

{namespace}/codegen/systems

to match tables

Reply:
or maybe we're already generating them that way and this isn't used anymore?

(on mobile so it's hard to tell)

Reply:
currently the `sytstemLibrariesDirectory` option is the name of the subdirectory in `codegen` at the namespace level: https://github.com/latticexyz/mud/blob/635233dc9bce159757bb28b412a3734a2b10aab5/packages/world/ts/node/render-solidity/worldgen.ts#L34-L42

Reply:
in that case I think this should be `systems`? because tables are libraries too and this makes it confusing

Reply:
done